skimage/color/tests/test_colorconv.py

```diff
@@ -438,6 +442,28 @@ def _get_lab0(self):
         rgb = img_as_float(self.img_rgb[:1, :1, :])
         return rgb2lab(rgb)[0, 0, :]
 
+    def test_yuv(self):
+        rgb = np.array([[[1.0, 1.0, 1.0]]])
+        assert_array_almost_equal(rgb2yuv(rgb), np.array([[[1, 0, 0]]]))
+        assert_array_almost_equal(rgb2yiq(rgb), np.array([[[1, 0, 0]]]))
+        assert_array_almost_equal(rgb2ypbpr(rgb), np.array([[[1, 0, 0]]]))
+        assert_array_almost_equal(rgb2ycbcr(rgb), np.array([[[235, 128, 128]]]))
```

Comment:
@soupault I'd love to, but it is hard to find reference values. In a sense the matrix values I put in are the most precise ones available, since they come directly from various standard documents like ITU BT.601 etc.   If you'd like I can add a test for what RGB [0,1,0] maps to.  Would that suffice?